### ISSUE TITLE
[Bug Report] Unknown custom element: <v-file-input>

### ISSUE THREAD

**Issue by vahdet** [vuetifyjs/vuetify#7825]
### Versions and Environment
**Vuetify:** 2.0.0-beta.7
**Vue:** 2.6.10
**Browsers:** Firefox 68.0
**OS:** Ubuntu undefined

### Steps to reproduce
- Follow [this guide](https://next.vuetifyjs.com/en/getting-started/quick-start) for installing Vuetify 2.0.0-beta.7 
- Use the new [`v-file-input`](https://next.vuetifyjs.com/en/components/file-inputs) tag somewhere.
- Watch the console

### Expected Behavior
Be able to use the file input field.

### Actual Behavior
It throws an error and does not display the file input
```
"[Vue warn]: Unknown custom element: <v-file-input> - did you register the component correctly? For recursive components, make sure to provide the 'name' option.
```

### Reproduction Link
<a href="https://codepen.io/vahdet/pen/mZZdjV" target="_blank">https://codepen.io/vahdet/pen/mZZdjV</a>

### Other comments
The pull request looks relevant https://github.com/vuetifyjs/vuetify/pull/7654 , is it on the way with the next release?

**Comment by DRoet** [comment]
yeah that PR isn't released yet in `Vuetify 2.0.0-beta.7`, you'll have to wait until the next release (maybe tomorrow)

**Comment by DRoet** [comment]
Your project is still using vuetify 1.x if that package.json is correct, you will have to upgrade to 2.x if you want to use v-file-input
